tests/conftest.py

```diff
@@ -126,7 +126,7 @@ def alternative_region():
 
 @pytest.fixture
 def signature_version():
-    return 's3'
+    return 'v4'
```

Comment:
this was critical to get the new ThreadedServer to recognize the correct service to route the request to